libs/shared/forgot-password-form/src/lib/shared-forgot-password-form.scss

```diff
@@ -1,7 +1,9 @@
-small {
-  font-size: 0.9rem !important;
-}
+.forgot-password-container {
+  small {
+    font-size: 0.9rem !important;
```

Comment:
I would prefer to avoid !important if possible. Write full class/elements path instead.